contracts/message-bus/MessageInbox.sol

```diff
@@ -75,20 +75,27 @@ contract MessageInbox is MessageBox, Proof {
     /**
      * @notice Setup message inbox.
      *
-     * @dev Function requires:
-     *          - inboundChannelIdentifier must be zero
-     *          - metachainId must not be zero
-     *          - messageOutbox address must not be zero
-     *          - verifyingAddress must not be zero
-     *          - stateRootProvider must not be zero
-     *
      * @param _metachainId Metachain identifier.
      * @param _messageOutbox MessageOutbox contract address.
      * @param _outboxStorageIndex Storage index of outbox mapping in
      *                            MessageOutbox contract.
      * @param _stateRootProvider State root provider contract address.
      * @param _maxStorageRootItems Defines how many storage roots should be
      *                             stored in circular buffer.
+     *
+     * \pre The function can be called only once.
+     * \pre `_metachainId` must not be zero.
+     * \pre `_messageOutbox` address must not be zero.
+     * \pre `_stateRootProvider` must not be zero.
+     *
+     * \post Sets `messageOutbox` storage variable with the value
+     *        of `_messageOutbox`.
+     * \post Sets `outboxStorageIndex` storage with the value
+     *       of `_outboxStorageIndex`.
+     * \post Sets `inboundChannelIdentifier` storage variable.
```

Comment:
What is your thought here. Do we need to clarify with which parameters it's called or it's obvious. If it's obvious we can skip them.

Reply:
I think parameters are obvious.